### PR TITLE
john-jumbo: re-enable test

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/john-jumbo.rb [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
-  version "1.8.0"
-  sha256 "bac93d025995a051f055adbd7ce2f1975676cac6c74a6c7a3ee4cfdd9c160923"
+
+  stable do
+    url "http://openwall.com/john/j/john-1.8.0-jumbo-1.tar.xz"
+    sha256 "bac93d025995a051f055adbd7ce2f1975676cac6c74a6c7a3ee4cfdd9c160923"
+    version "1.8.0"
+
+    patch do
+      url "https://github.com/magnumripper/JohnTheRipper/commit/d29ad8aabaa9726eb08f440001c37611fa072e0c.diff"
```
Comment (mikemcquaid) [inline Library/Formula/john-jumbo.rb]: Can you please submit this patch to the upstream developers of this project (if it hasn't already) and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file :wink: . Thanks!

**Comment by mikemcquaid** [comment]
:+1: other than comment

File: Library/Formula/john-jumbo.rb (hunk 1 above)
Comment (dunn) [inline Library/Formula/john-jumbo.rb]: It's an upstream commit!

Comment (dunn) [inline Library/Formula/john-jumbo.rb]: But I'll add a comment.

**Comment by DomT4** [comment]
You may want to also pull the bottle here, so the `brew test` acts identically for newly-poured bottled builds as it does from-source builds.

**Comment by dunn** [comment]
Good point; will do.

**Comment by dunn** [comment]
That's why Travis failed too https://travis-ci.org/Homebrew/homebrew/jobs/101271480

**Comment by DomT4** [comment]
Ah, yeah. The follow-up bottle [job](https://travis-ci.org/Homebrew/homebrew/builds/101272700) should pass, theoretically.

**Pull request closed without merging**